python_modules/dagster/dagster/_core/storage/asset_value_loader.py

```diff
@@ -89,13 +91,15 @@ def load_asset_value(
         io_manager_key = assets_def.get_io_manager_key_for_asset_key(asset_key)
         io_manager_def = resource_defs[io_manager_key]
         required_resource_keys = io_manager_def.required_resource_keys | {io_manager_key}
-
+        config = config or {}
+        resource_config = config.get('resources', {})
```

Comment:
it seems like the only thing we're pulling off of the config here is the resources config. I wonder if we would be better served by just making the parameter called `resources_config` directly, and allowing you to configure each resource by key without having to specify the top level `resources` key to the config dictionary.

Reply:
I am passing the config into `build_input_context`, is it not required there?

Reply:
As well, I tend to pass the same config around with config for the whole project, so it's a little less boiler plate and simplifies things a bit

Reply:
The config argument on `build_input_context` is for input config specifically, so the config argument provided here wouldn't match up. `build_input_context` also has a `resource_config` argument (should probably be called resources_config) that allows for providing config for the resources.

Regarding passing the same config around, that's definitely reasonable and I imagine many users do the same thing, but it seems like the difference is just between passing `run_config` and `run_config.get("resources")`, no? In which case, I'm inclined to scope it down to just resources_config to keep the data model simpler.

Reply:
OK sounds good, I have updated accordingly